src/org/plovr/ConfigOption.java

```diff
@@ -272,6 +272,16 @@ public void apply(String outputCharset, Config.Builder builder) {
     }
   }),
 
+
+  CACHE_OUTPUT_FILE("cache-output-file", new ConfigUpdater() {
```

Comment:
maybe the answer is that it uses a tmp file by default, but allows you to set a cache-output-file if you care

or maybe cache-output-file is useful for cases where you plan to start and stop plovr but want the cache to persist.